relay-server/src/actors/store.rs

```diff
@@ -1049,3 +1103,162 @@ impl Message for KafkaMessage {
 fn is_slow_item(item: &Item) -> bool {
     item.ty() == &ItemType::Attachment || item.ty() == &ItemType::UserReport
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use relay_common::ProjectKey;
+
+    /// Helper function to get the arguments for the `fn extract_kafka_messages(...)` method.
+    fn test_arguments_extract_kafka_msgs() -> (Instant, EventId, Scoping, Vec<ChunkedAttachment>) {
```

Comment:
nit: To me, the `test_` prefix suggests that the function itself is a test.

```suggestion
    fn  arguments_extract_kafka_msgs() -> (Instant, EventId, Scoping, Vec<ChunkedAttachment>) {
```